pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -468,3 +450,17 @@ func ConnectClickHouse(url string) (*sql.DB, error) {
 	}
 	return connect, nil
 }
+
+func parseDatabaseURL(dbUrl string) (protocol, string, error) {
+	u, err := url.Parse(dbUrl)
+	if err != nil {
+		return protocolUnknown, "", fmt.Errorf("failed to parse ClickHouse database URL: %v", err)
+	}
+	proto := u.Scheme
```

Comment:
let's give an error if `u.Path`, `u.RawQuery` or `u.UserInfo` is set.
when we do support `http` as the scheme, we will need to support `u.Path`

Reply:
make sense, done